client/src/components/Navbar.js

```diff
@@ -46,6 +46,7 @@ const NavbarStyles = makeStyles({
 });
 
 const Navbar = () => {
+  const isProd = process.env.REACT_APP_ENV === "production";
```

Comment:
This adds a new environment variable, we can maybe use NODE_ENV itself that React seems to use.

Source: https://medium.com/@tacomanator/environments-with-create-react-app-7b645312c09d